tests/dataset_builders/pie/test_aae2.py

```diff
@@ -0,0 +1,612 @@
+        "'we', 'always', 'find', 'that', 'we', 'need', 'the', 'cooperation')"
+    )
+    assert (
+        str(sorted_entities[8])
+        == "('take', 'olympic', 'games', 'which', 'is', 'a', 'form', 'of', 'competition', 'for', 'instance', ',', "
+        "'it', 'is', 'hard', 'to', 'imagine', 'how', 'an', 'athlete', 'could', 'win', 'the', 'game', 'without', "
+        "'the', 'training', 'of', 'his', 'or', 'her', 'coach', ',', 'and', 'the', 'help', 'of', 'other', "
+        "'professional', 'staff', '##s', 'such', 'as', 'the', 'people', 'who', 'take', 'care', 'of', 'his', "
+        "'diet', ',', 'and', 'those', 'who', 'are', 'in', 'charge', 'of', 'the', 'medical', 'care')"
+    )
+    assert (
+        str(sorted_entities[9])
+        == "('without', 'the', 'cooperation', ',', 'there', 'would', 'be', 'no', 'victory', 'of', 'competition')"
+    )
+    assert (
+        str(sorted_entities[10])
+        == "('a', 'more', 'cooperative', 'attitudes', 'towards', 'life', 'is', 'more', 'profitable', "
+        "'in', 'one', \"'\", 's', 'success')"
+    )
+
+
+def test_tokenized_documents_with_entities_and_relations_all(
+    dataset_of_text_documents_with_labeled_spans_and_binary_relations, tokenizer, dataset_variant
+):
+    for (
+        split,
+        docs,
+    ) in dataset_of_text_documents_with_labeled_spans_and_binary_relations.items():
+        for doc in docs:
+            # Note, that this is a list of documents, because the document may be split into chunks
+            # if the input text is too long.
+            tokenized_docs = tokenize_document(
+                doc,
+                tokenizer=tokenizer,
+                return_overflowing_tokens=True,
+                result_document_type=TestTokenDocumentWithLabeledSpansAndBinaryRelations,
+                strict_span_conversion=True,
+                verbose=True,
+            )
+            # we just ensure that we get at least one tokenized document
+            assert tokenized_docs is not None
+            assert len(tokenized_docs) > 0
+
+
+@pytest.fixture(scope="module")
+def tokenized_documents_with_labeled_spans_binary_relations_and_labeled_partitions(
+    dataset_of_text_documents_with_labeled_spans_binary_relations_and_labeled_partitions, tokenizer
+) -> List[TestTokenDocumentWithLabeledSpansBinaryRelationsAndLabeledPartitions]:
+    # get a document to check
+    doc = dataset_of_text_documents_with_labeled_spans_binary_relations_and_labeled_partitions[
+        "train"
+    ][0]
+    # Note, that this is a list of documents, because the document may be split into chunks
+    # if the input text is too long.
+    tokenized_docs = tokenize_document(
+        doc,
+        tokenizer=tokenizer,
+        partition_layer="labeled_partitions",
+        return_overflowing_tokens=True,
+        result_document_type=TestTokenDocumentWithLabeledSpansBinaryRelationsAndLabeledPartitions,
+        strict_span_conversion=False,
```

Comment:
I changed `strict_span_conversion=True` in 491c137; however, there seemed to be missing annotations. Will need to investigate further

Reply:
Update: 

when documents are partitioned into paragraphs, we set `strict_span_conversion` to `False`. Otherwise, it loses the annotations from relations that contain spans from other paragraphs and produces an error.

Note that: these inter-paragraph relations did not exist in the original dataset, but they were added when converting `BratDocumentWithMergedSpans` to `TextDocument...`, in order to link `Claim`'s (in the body paragraphs) to `MajorClaims`'s (in intro/conclusion paragraphs) and also to link `MajorClaim`'s to one another.